src/main/java/tc/oc/pgm/PGMImpl.java

```diff
@@ -332,6 +333,13 @@ public Identity getIdentity(UUID playerId, String username, @Nullable String nic
     if (Config.AutoRestart.enabled()) {
       getServer().getScheduler().runTaskTimer(this, new ShouldRestartTask(this), 0, 20 * 60);
     }
+    if (Config.Discord.enabled()) {
+      try {
+        DiscordClient.initialize();
+      } catch (Exception e) {
+        logger.severe(e.getMessage());
```

Comment:
Typically we use the `#log` method, also use a warning. `logger.log(Level.WARN, "Unable to initialize Discord client", e);`